lighthouse-core/runner.js

```diff
@@ -218,7 +218,7 @@ class Runner {
    */
   static async _runAudit(auditDefn, artifacts, settings, runWarnings) {
     const audit = auditDefn.implementation;
-    const status = `Evaluating: ${audit.meta.title}`;
+    const status = `Evaluating: ${i18n.getDisplayValue(audit.meta.title)}`;
```

Comment:
since `displayValue` is a thing i dont think we should reuse that name for this.

`getOriginalMessage` ?

Reply:
yeah I hesitated here, original message isn't quite right either. how bout 

`getFormatted`
`toFormatted`
`getFormattedValue`
`getFormattedString`
`toFormattedString`